Source/Sandbox/SandboxGame.cs

```diff
@@ -86,59 +93,18 @@ protected override void UnloadContent()
 
         protected override void Update(GameTime gameTime)
         {
-            var deltaTime = (float) gameTime.ElapsedGameTime.TotalSeconds;
+            _eventDrivenInput.Update(gameTime);
+
+            var deltaTime = (float) gameTime.ElapsedGameTime.TotalMilliseconds * .001f;
```

Comment:
@ClassicThunder Out of curiosity, why did you use `TotalMilliseconds` vs `TotalSeconds`?